test/system/030-run.bats

```diff
@@ -1120,10 +1120,12 @@ EOF
     CONTAINERS_CONF_OVERRIDE="$containersconf" run_podman run --rm --read-only=false $IMAGE touch /testrw
     CONTAINERS_CONF_OVERRIDE="$containersconf" run_podman run --rm $IMAGE touch /tmp/testrw
     for dir in /tmp /var/tmp /dev /dev/shm /run; do
-        CONTAINERS_CONF_OVERRIDE="$containersconf" run_podman 1 run --rm --read-only-tmpfs=false $IMAGE touch $dir/testro
```

Comment:
These are such confusing options. I don't have the brainpower to review right now, so just two quick points:
* for those who care, this is a breaking change
* for completeness, should we check all permutations of `--read-only` with `--read-only-tmpfs`?

Reply:
The breaking change would have been to change the --read-only-tmpfs to --read-write-tmpf, which it is called internally.  The issue is that if the user actually sets the constant it does it backwards, While if the user never sets it the default is to have read-write-tmpfs.

Reply:
Bottom line is the bug --read-only-tmpfs=false Was turning on read-only-tmpfs while --read-only-tmfs=true was turning off read-only-tmpfs.